packages/core/src/llm/LLM.ts

```diff
@@ -674,21 +673,13 @@ export class Anthropic extends BaseLLM {
     };
   }
 
-  mapMessagesToPrompt(messages: ChatMessage[]) {
```

Comment:
We need to support both the "legacy" models as well as Claude 3, so need to think about how to make this work for both.